buildSrc/src/main/java/Dependencies.kt

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 object Versions {
-    const val kotlin = "1.5.20"
+    const val kotlin = "1.5.10"
```

Comment:
I needed to downgrade from 1.5.20 to 1.5.10 because the Compose compiler plugin was complaining. I assume that with the next release of Compose we can go up again.